queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon'
+SELECT * FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '01-01-2019'
+SELECT name FROM animals WHERE neutered=TRUE AND escape_attempts < 3
+SELECT date_of_birth from animals WHERE name='Agumon' OR name='Pikachu'
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5
+SELECT * from animals WHERE neutered=TRUE
+SELECT * from animals WHERE name <> 'Gabumon'
+SELECT * from animals WHERE weight_kg >=10.4 and weight_kg <= 17.3
```

Comment:
- [ ] Kindly include `;` at the end of each query to end it, It causes errors when we do not end queries with `;`.

- [ ] Line 4 should be WHERE `date_of_birth BETWEEN '2016/01/01' AND '2019/12/31'` so that you get the names of all animals born between 2016 and 2019.